jobmon_gui/src/workflow_overview_page/workflow_overview.tsx

```diff
@@ -174,13 +178,12 @@ function App() {
             </Form.Group>
           </Row>
 
-          <Row className="mb-3">
-            <Col className='mt-4'>
-              <Button variant="primary" type="submit" className="btn btn-custom mx-40p" onClick={onSubmit}>
-                Submit
-              </Button>
-            </Col>
-          </Row>
+          <div className="text-center">
+            <div className="btn-toolbar d-inline-block">
+              <button type="submit" className="btn btn-custom mr-1" onClick={onSubmit}>Submit</button>
```

Comment:
for my own education what's the difference between <Row><Col><Button> and <div><div><button> functionally?

Reply:
Github isn't showing what two things you're asking about the difference about. Imma assume you're asking about the difference between having this display with <Row> vs <div>. Rows are a part of the bootstrap grid system. It's possible to do what I wanted to do (two center buttons with a small amount of space between them) with Rows/Cols, but it was slightly more annoying to get the grid to play nicely than just using a div. So this was mostly a personal preference change.